### DISCUSSION TITLE
Uploading templates

### DISCUSSION THREAD

**Discussion by MarCorrado** [freeplane/freeplane#279] [Help]
If I have created a template, how can I share it with the community?
Thanks

**Comment by rexelbartolome** [comment]
I think you can post them here https://github.com/freeplane/freeplane/discussions/254

Just look for the .mm file that you made as a template and upload it there :) a pic of what it looks like/what it does would be great to pair it with!

**Reply by MarCorrado** [reply]
Thank you. I made as suggested.
